rio-orphans/src/RIO/Orphans.hs

```diff
@@ -35,6 +39,23 @@ instance MonadBaseControl IO (RIO env) where
   liftBaseWith = withRunInIO
   restoreM = return
 
+-- | @since 0.1.3.0
```

Comment:
Looks like these version numbers should be 0.1.1.0 instead.

Reply:
Good catch 👀